### PR TITLE
Insertion of sonar msgs

### PR DESCRIPTION
**Pull request by ivan1993br**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+## Compile as C++11, supported in ROS Kinetic and newer
+# add_compile_options(-std=c++11)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
+   roscpp
+   rospy
+   std_msgs
```
Comment (g-arjones) [inline CMakeLists.txt:11]: Does this require roscpp, rospy and std_msgs?

Comment (ivan1993br) [reply CMakeLists.txt:11]: No

Comment (g-arjones) [reply CMakeLists.txt:11]: What about std_msgs?

Comment (ivan1993br) [reply CMakeLists.txt:11]: Yes because of the header type on SonarStamped, it is implemented on the std_msgs

File: CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  CATKIN_DEPENDS message_runtime
+  )
+
+###########
+## Build ##
+###########
+
+## Specify additional locations of header files
+## Your package locations should be listed before other locations
+include_directories(
```
Comment (g-arjones) [inline CMakeLists.txt]: This is doing nothing

File: CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+# include
+# ${catkin_INCLUDE_DIRS}
+)
+
+
+#############
+## Install ##
+#############
+
+# Install urdf files
```
Comment (g-arjones) [inline CMakeLists.txt]: What urdf files?

File: CMakeLists.txt (hunk 1 above)
Comment (g-arjones) [reply CMakeLists.txt:11]: Makes sense.. Should std_msgs be in package.xml in this case?

**Comment by g-arjones** [comment]
@romulogcerqueira I didn't know Ivan was expecting you comments as well.. Sorry about that. Could you take a look, please?

**Pull request merged**